components/list/test/list-item-drag-handle.visual-diff.js

```diff
@@ -0,0 +1,44 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-list-item-drag-handle', () => {
+
+	const visualDiff = new VisualDiff('list-item-drag-handle', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/components/list/test/list-item-drag-handle.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(async() => await browser.close());
+
+	describe('dragger', () => {
+		it('keyboard-mode', async function() {
+			const rect = await visualDiff.getRect(page, '#drag-handle-keyboard-active');
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+
+		it('simple', async function() {
+			const rect = await visualDiff.getRect(page, '#drag-handle');
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+		it('focus', async function() {
+			await visualDiff.resetFocus(page);
+			await focusMethod('d2l-list-item-drag-handle');
```

Comment:
This could be... `await page.$eval('d2l-list-item-drag-handle', item => item.focus());`.

Not a big deal either way... 🤷‍♂️

Reply:
Kept it since refactoring the `keyboardActive` to private I used the `focusMethod` in another spot.